mail_factory/models.py

```diff
@@ -2,9 +2,14 @@
 
 import django
 from django.conf import settings
-from django.utils.importlib import import_module
 from django.utils.module_loading import module_has_submodule
 
+try:
+    from importlib import import_module
+except ImportError:
+    # Compatibility for python-2.6
+    from django.utils.importlib import import_module
```

Comment:
Python2.6 is still tested with tox. :confused: 
Do you think we should drop python2.6 and Django related versions?